pydav/wsgidav.yaml

```diff
@@ -0,0 +1,85 @@
+# Documentation: https://wsgidav.readthedocs.io/en/latest/user_guide_configure.html#sample-wsgidav-yaml
+
+server: cheroot
+
+# Server hostname (default: localhost, use --host on command line)
+host: 0.0.0.0
+
+# Server port (default: 8080, use --port on command line)
+port: 8080
+
+# ----------------------------------------------------------------------------
+#: Modify to customize the WSGI application stack.
+#: See here for an example how to add custom middlewares:
+#:   https://wsgidav.readthedocs.io/en/latest/user_guide_configure.html#middleware-stack
+middleware_stack:
+    - wsgidav.mw.cors.Cors
+    # - wsgidav.mw.debug_filter.WsgiDavDebugFilter
+    - wsgidav.error_printer.ErrorPrinter
+    - wsgidav.http_authenticator.HTTPAuthenticator
+    - wsgidav.dir_browser.WsgiDavDirBrowser
+    - wsgidav.request_resolver.RequestResolver  # this must be the last middleware item
+
+# ==============================================================================
+# AUTHENTICATION
+
+http_authenticator:
+    #: Allow basic authentication
+    accept_basic: true
+    #: Allow digest authentication
+    accept_digest: true
+    #: true (default digest) or false (default basic)
+    default_to_digest: true
+    #: Header field that will be accepted as authorized user.
+    #: Including quotes, for example: trusted_auth_header = 'REMOTE_USER'
+    trusted_auth_header: null
+    #: Domain controller that is used to resolve realms and authorization.
+    #: Default null: which uses SimpleDomainController and the
+    #: `simple_dc.user_mapping` option below.
+    #: (See http://wsgidav.readthedocs.io/en/latest/user_guide_configure.html
+    #: for details.)
+    domain_controller: wsgidav.dc.simple_dc.SimpleDomainController
+    # domain_controller: wsgidav.dc.simple_dc.SimpleDomainController
+    # domain_controller: wsgidav.dc.pam_dc.PAMDomainController
+    # domain_controller: wsgidav.dc.nt_dc.NTDomainController
+
+# ----------------------------------------------------------------------------
+# CORS
+# (Requires `wsgidav.mw.cors.Cors`, which is enabled by default.)
+cors:
+    #: List of allowed Origins or '*'
+    #: Default: false, i.e. prevent CORS
+    allow_origin: false
+
+# ==============================================================================
+# DEBUGGING
+
+#: Set verbosity level (can be overridden by -v or -q arguments)
+verbose: 3
+
+logging:
+    #: Enable logging when using wsgidav in library mode (always on, when running as CLI)
+    enable: true
+
+# ----------------------------------------------------------------------------
+# WsgiDavDirBrowser
+dir_browser:
+    enable: false
+
+# ==============================================================================
+# SHARES
+provider_mapping:
```

Comment:
provider_mapping & simple_dc should be configurable from external. As envs or config file. Ensure KISS !!!